pkg/gateway/lb/handler/grpc/option_test.go

```diff
@@ -1,18 +0,0 @@
-//
```

Comment:
same as above comment, we should leave this file to include the implementation to code coverage.

Reply:
There is no test in this file so I deleted it.
And if test template update we can re-generate it.